custom-namespaces.yaml

```diff
@@ -1062,6 +1062,18 @@ casa:
       type: table_view
       tables:
         - table: mozdata.casa.users
+reference:
+  glean_app: false
+  connection: bigquery-oauth
+  pretty_name: Reference
+  owners:
+    - cmorales@mozilla.com
+  spoke: looker-spoke-default
```

Comment:
if it's bigquery-oauth connection in has to go into looker-spoke-private

Reply:
Thanks! I realized I don't the oauth connection